src/elfpy/pricing_models/yieldspace.py

```diff
@@ -323,7 +325,7 @@ def calc_in_given_out(
             and the unit of the tokens).
         market_state : MarketState
             The state of the AMM's reserves and share prices.
-        fee_percent : float
+        trade_fee_percent : float
             The percentage of the difference between the amount paid without
             slippage and the amount received that will be added to the input
             as a fee.
```

Comment:
Can you add `redemption_fee_percent` to the docstring pls?

Reply:
yup!